client/src/app/app.ts

```diff
@@ -17,7 +17,7 @@ import { ManagingBodyService } from './managing-body/managing-body.service';
 @RouteConfig([
   { path: '/home',               component: HomeComponent,                name: 'Home', useAsDefault: true },
```

Comment:
I didn't wrote it in the original issue, but because we change the files, on the way, let's write each properties in a new line. Example:
```
{
  path: '/home',
  component: HomeComponent,
  name: 'Home',
  useAsDefault: true
},
```